static/js/subs.js

```diff
@@ -1054,10 +1054,9 @@ $(function () {
 
         loading.make_indicator(indicator_elem);
 
-        channel.post({
-            url: "/json/get_subscribers",
+        channel.get({
+            url: "/json/streams" + encodeURIComponent(stream) + "/members",
```

Comment:
@timabbott , We can use encodeURIComponent to encode the stream name but I am worrying about decoding of the url, how can we decode this name in the get_subscriber_backend logic??